### ISSUE TITLE
Not working.

### ISSUE THREAD

**Issue by plavanabond** [Alikhalkhali/programs-watcher#10]
Not receiving any updates since last week. Any issue with the app?

**Comment by Money-1** [comment]
Yep is literally not working anymore I thought I had some issues. After restarting , delete and redownload its still the same. My last updated was on 21st of august 2024. That's it no more updates.

**Comment by Sh4d0wHunt3rX** [comment]
It's because this script get data from somewhere else which has bug:

https://github.com/Osb0rn3/bugbounty-targets/issues/6

**Comment by abdollahianali** [comment]
there is no in_scope in target-Object.
to fix that :
`    if program:  
        for target in program.get("target_groups", []):
            if target.get("in_scope") is False:
                for item in target.get("targets", []):
                    dataJson["outOfScope"].append(item.get("name", "Unknown Target"))
            else:
                for item in target.get("targets", []):
                    dataJson["inScope"].append(item.get("name", "Unknown Target"))
`
in bugcrowd.py
